.ci/pitest-suppressions/pitest-misc-suppressions.xml

```diff
@@ -207,15 +207,6 @@
     <lineContent>getId(),</lineContent>
   </mutation>
 
-  <mutation unstable="false">
-    <sourceFile>TranslationCheck.java</sourceFile>
-    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.TranslationCheck</mutatedClass>
-    <mutatedMethod>validateUserSpecifiedLanguageCodes</mutatedMethod>
-    <mutator>org.pitest.mutationtest.engine.gregor.mutators.NonVoidMethodCallMutator</mutator>
-    <description>removed call to com/puppycrawl/tools/checkstyle/checks/TranslationCheck::getId</description>
-    <lineContent>WRONG_LANGUAGE_CODE_KEY, new Object[] {code}, getId(), getClass(), null);</lineContent>
```

Comment:
> Any fields dropped when passing to the constructor were not being used anyways. So we basically get the same effect, with less unnecessary coding.

This shows what I am talking about.

`getId` is never used as part of the exception message as we just provide the localized text of the message and the id is not included in it. ID is needed for violations when printing audit events only, which this is never used for.